### DISCUSSION TITLE
Contributor Call: 21 January 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3393] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 21 January 2023 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jan-21-2023/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
TBD

> Notes will be added after the call.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark 
- @lexander2002 
- @KaerMorhan 
- @bobgeorgethe3rd 
- @SeaZeeZee 
- @BenJamesBen 
- @darigovresearch 

✨ **A mini-newsletter on the FS website about happenings on Discord**
Post-it-weekly “What’s happening on Discord” but it would be more of a manual process at the moment.

_Bob: I think it would be manual at the moment until we will come up with a way of automating it. Contributor role is limiting posting abilities. Do we want to limit access to posting to make sure nothing goes on the website that we don’t want to?_
_Ben: I think it would be nice to have a small editorial team. Like a mini-newsletter. Maybe it can be in just fragments and not written like a complete blog post._
_Bob: An interesting point about how it will work with translations._
_Karen: Yes, right now blogposts are not translated. And I think Joost was bringing up something about not translating blogposts and just making different blogposts in different languages._

✨ **Animal bags/backpacks/sleeping bags: Hi and possibly Octoplushy**
A discussion in [general](https://discord.com/channels/698854858052075530/698854858496802868) channel.

✨ **Bucket hat update (Bob)**
It’s on a backburner, hats are not in my priority right now. I will need to review the code because I know it can be better. I don’t want to put a version that I know will rewrite.

✨ **Jane and Stede (Zee)**
It’s also a bit on hold because _**reasons**_.

_Karen: It’s okay, I also have a pattern that sits there and gathers dust._
_Zee: I am actually getting to finish my 1780s stays. They’re almost done. I will think about coding a pair of short stays. They’re comparatively easy since they don’t accentuate your waist and don’t use negative ease._
_Ben: I was working on a pirate shirt. I have a full pattern in code but I didn’t test it. I was focused on documentation instead._
_Karen: I might help with testing._

✨ **Karen**
I’m excited for the next wave of showcases. I did an Aaron dress and Popeye did a dropped shoulder Sven with very nice instructions. It will have a lot of cool stuff. I’m doing them backwards at the moment.
_Zee: Maybe I can help so you don’t have to do it all by yourself._

✨ **Bob**
Someone is going to tackle Jeager documentation and I’m very excited for it. Carlton and Carlita also need documentation. I will do them eventually but if somebody is up for writing it – please go ahead.

_Zee: Everybody who does documentation, please tag somebody other than Joost for proofreading._
_Ben: Do we need a separate documentation channel?_
_Bob: I don’t think we need a separate channel, we have tags and roles for it._
_Darigov Research said you can tag them for proofreading._

✨ **Ben**
Basically, I’m documenting a technical coding guide and a style guide about how to write documentation for FS.

_Zee: I saw your guide, I have some thoughts about it (transfer to the next CC agenda). It should definitely be more of a guide than a hard set of rules. Plus, we have proofreaders for a lot of stuff._
_Karen: You can always post documentation and mention that it needs an edit because other people occasionally edit it anyway._

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
